OutOfSchool/OutOfSchool.BusinessLogic/Models/CompetitiveEvent/CompetitiveEventCreateDto.cs

```diff
@@ -0,0 +1,105 @@
+﻿//using OutOfSchool.Common.Enums;
+using OutOfSchool.Common.Enums;
+using OutOfSchool.Services.Enums;
+using System.ComponentModel.DataAnnotations;
+
+namespace OutOfSchool.BusinessLogic.Models.CompetitiveEvent;
+public class CompetitiveEventCreateDto
+{
+    [DataType(DataType.Text)]
+    [MaxLength(250)]
+    [MinLength(1)]
+    public string Title { get; set; }
+
+    [Required(ErrorMessage = "ShortTitle is required")]
+    [DataType(DataType.Text)]
+    [MaxLength(100)]
+    [MinLength(1)]
+    public string ShortTitle { get; set; }
+
+    [Required]
+    [EnumDataType(typeof(CompetitiveEventStates), ErrorMessage = Constants.EnumErrorMessage)]
+    public CompetitiveEventStates State { get; set; } = CompetitiveEventStates.Draft;
+
+    public DateTimeOffset RegistrationStartTime { get; set; }
+
+    public DateTimeOffset RegistrationEndTime { get; set; }
+
+    public Guid? ParentId { get; set; } = null;
+
+    public Guid? BuildingHoldingId { get; set; }
+
+    public Guid? ChildParticipantId { get; set; }
+
+    public List<CompetitiveEventDescriptionItemDto> CompetitiveEventDescriptionItems { get; set; }
+
+    [MaxLength(2000)]
+    public string AdditionalDescription { get; set; }
+
+    [Required]
+    public DateTimeOffset ScheduledStartTime { get; set; }
+
+    [Required]
+    public DateTimeOffset ScheduledEndTime { get; set; }
+
+    [Required]
+    public uint NumberOfSeats { get; set; } = uint.MaxValue;
+
+    [Required]
+    public int CompetitiveEventAccountingTypeId { get; set; }
+
+    [MaxLength(2000)]
+    public string Description { get; set; }
+
+    [MaxLength(2000)]
+    public string DescriptionOfTheEnrollmentProcedure { get; set; }
+
+    public Guid? OrganizerOfTheEventId { get; set; }
```

Comment:
OrganizerOfTheEventId is nullable because if it's null, it indicates that the event organizer is the provider-owner. However, in the future, we may need to support cases where the organizer is a different organization than the one that created the event.